<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="title_lbl">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ustawienia wtyczki APP</string>
       </property>
       <property name="margin">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="empty_lbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QScrollArea" name="settings_scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>676</width>
            <height>324</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QgsCollapsibleGroupBox" name="verticalGroupBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="title">
              <string>Domyślna ścieżka zapisu plików</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="collapsed">
              <bool>false</bool>
             </property>
             <property name="saveCollapsedState">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="folder_btn">
                <property name="text">
                 <string>Wybierz katalog</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="folder_lbl">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QgsCollapsibleGroupBox" name="verticalGroupBox_2">
             <property name="title">
              <string>Konfiguracja identyfiktora zbioru danych przestrzennych</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="2" column="2">
                 <widget class="QLabel" name="label_3">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Identyfikator jednostki podziału terytorialnego, dla której prowadzony jest zbiór danych przestrzennych, utworzony na podstawie identyfikatora z rejestru TERYT, bez ostatniego członu określającego rodzaj jednostki.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>../../../../img/info2.png</pixmap>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="jpt_lbl">
                  <property name="text">
                   <string>JPT</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Identyfikator lokalny APP</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="edycjaILAPP">
                  <property name="placeholderText">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="4" column="2">
                 <widget class="QLabel" name="label_5">
                  <property name="toolTip">
                   <string>Identyfikator lokalny identyfikujący jednoznacznie i unikalnie akt planowania przestrzennego w zbiorze danych przestrzennych.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>../../../../img/info2.png</pixmap>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="rodzajZbioru_comboBox"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="rodzajZbioru_lbl">
                  <property name="text">
                   <string>Rodzaj zbioru</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Skrócona nazwa typu aktu planowania przestrzennego.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>../../../../img/info2.png</pixmap>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="przestrzenNazw_lbl">
                  <property name="text">
                   <string>Przestrzeń nazw APP</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="numerZbioru_lbl">
                  <property name="text">
                   <string>Numer zbioru</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QgsFilterLineEdit" name="przestrzenNazw_lineEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="tabletTracking">
                   <bool>true</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="showClearButton">
                   <bool>true</bool>
                  </property>
                  <property name="showSearchIcon">
                   <bool>false</bool>
                  </property>
                  <property name="showSpinner">
                   <bool>false</bool>
                  </property>
                  <property name="qgisRelation" stdset="0">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QgsFilterLineEdit" name="numerZbioru_lineEdit">
                  <property name="qgisRelation" stdset="0">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_1">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Przestrzeń nazw identyfikująca w sposób jednoznaczny źródło danych obiektu. Wartość atrybutu przestrzeń nazw powinna jednoznacznie identyfikować zbiór danych przestrzennych, do którego należy instancja typu obiektu.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>../../../../img/info2.png</pixmap>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QLabel" name="label_2">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Unikalny numer, który stanowi element przestrzeni nazw Identyfikatora zbioru danych przestrzennych (§ 4 ust.1 pkt 1 lit. c Rozporządzenia) – numer ten nadaje Główny Geodeta Kraju.</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>../../../../img/info2.png</pixmap>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="empty2_lbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_btn">
         <property name="text">
          <string>Zapisz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_btn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Zamknij</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>settings_scrollArea</tabstop>
  <tabstop>verticalGroupBox</tabstop>
  <tabstop>folder_btn</tabstop>
  <tabstop>verticalGroupBox_2</tabstop>
  <tabstop>edycjaILAPP</tabstop>
  <tabstop>verticalGroupBox</tabstop>
  <tabstop>verticalGroupBox</tabstop>
  <tabstop>save_btn</tabstop>
  <tabstop>exit_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
